Warning dialog box for analytics disable scenario

ilkayilknur opened the pull request:
This PR is related with https://xomni.visualstudio.com/web/wi.aspx?pcguid=8028f4c2-f298-4101-ad0f-e137df8b18a1&id=5468.

We upgraded dev branch to  v4.0 XOMNI APIs last week. So you need to run XOMNI 4.0 APIs and Old CMS on your machine and test this page. (I will cherry-pick this commit then deploy this commit to staging &prod environments when I get approval for this PR.)

Here is the screenshot of the dialog box for quick review.
![capture4](https://cloud.githubusercontent.com/assets/5785927/13457032/a248bee8-e06e-11e5-86f5-bc5595dbfc05.PNG)

@daronyondem can you take a look please ?

daronyondem commented:
Let's add a space before and after ampersand. Please add the sentence below to the end of the warning as well.

¨This process is irreversible.¨

Thanks.

ilkayilknur commented:
I've added additional warning message and fixed the spaces. 
Screenshot
![capture4](https://cloud.githubusercontent.com/assets/5785927/13460052/b9fedd8a-e080-11e5-9a3a-0f44bae791ca.PNG)
